src/infrastructure/transaction/NamespaceMosaicIdGenerator.ts

```diff
@@ -36,27 +36,35 @@ export class NamespaceMosaicIdGenerator {
      * @param {string} namespaceName - The namespace name
      * @returns sub namespace id
      */
-    public static namespaceId = (namespaceName) => {
-        const path = IdGenerator.generateNamespacePath(namespaceName);
-        return path.length ? IdGenerator.generateNamespacePath(namespaceName)[path.length - 1] : [];
+    public static namespaceId = (namespaceName, signSchema: SignSchema = SignSchema.SHA3) => {
```

Comment:
Remove the default value. Use NetworkType

Reply:
this is from first commit, is outdated..